.github/workflows/ci-pr-label.yml

```diff
@@ -0,0 +1,24 @@
+name: CI (PR Label)
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Set PR Labels
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+    steps:
+        - uses: actions/labeler@v4
+          with:
+            repo-token: ${{ secrets.GITHUB_TOKEN }}
+            sync-labels: true
+            dot: true
```

Comment:
This workflow behaves based on mapping or labels and paths defined in `.github/labeler.yml`.